src/main/java/edu/oregonstate/cs361/battleship/HardBattleshipModel.java

```diff
@@ -79,23 +87,93 @@ else if (orient == 1) {                       // if vert, row restricted by leng
             }
         }
 
-        /* System.out.println("Ship locations after random placement: ");
+        /*System.out.println("Ship locations after random placement: ");
         for (Ship ship: comp_ships) {
             System.out.println(ship.getName() + ": starts " + ship.start.getAcross() + ", " + ship.start.getDown() + "; ends " + ship.end.getAcross() + ", " + ship.end.getDown());
         }
         System.out.println(); */
```

Comment:
@kbajno actually, I'm using these as "confirmation" for the hard mode user story, since they live in the default constructor and we don't have any tests for it.